docs/builders/index.mdx

```diff
@@ -26,3 +26,21 @@ the following VMware builders:
   an existing VMware VM you want to use as the source. As an additional
   benefit, you can feed the artifact of this builder back into Packer to
   iterate on a machine.
+
+## VMware (Workstation) Player on Linux
+
+Unfortunately just installing VMware Player is not quite enough to be able to
+use packer. You will also need the `qemu-img` command which is available in
+the `qemu` package on Debian- and Fedora-like systems.
+
+Additionally you will need to have the `vmrun` command, which is part of the
+VMware [Virtual Infrastructure eXtension][vix-api] [(VIX) SDK][vix-sdk].
+
+Finally, if you installed a version of VMware player newer than 7.0, you must
```

Comment:
I would remove the versioning. Ideally the plugin should support only those versions supported by VMware per the lifecycle matrix. See project README.